DataFormats/TotemRawData/src/classes_def.xml

```diff
@@ -0,0 +1,31 @@
+<lcgdict>
+  <class name="TotemRawEvent" ClassVersion="6">
+    <version ClassVersion="6" checksum="2006209735"/>
+  </class>
+  <class name="edm::Wrapper<TotemRawEvent>"/>
+
+  <class name="TotemRawEvent::OptoRxMetaData" ClassVersion="2">
+    <version ClassVersion="2" checksum="191520249"/>
+  </class>
+
+  <class name="TotemRawEvent::TriggerData" ClassVersion="2">
+    <version ClassVersion="2" checksum="2680259756"/>
+  </class>
+
+  <class name="TotemStructuralVFATId" ClassVersion="2">
+    <version ClassVersion="2" checksum="2790056223"/>
+  </class>
+
+  <class name="TotemRawToDigiStatus" ClassVersion="6">
+    <version ClassVersion="6" checksum="3446987415"/>
```

Comment:
@jan-kaspar 
looking at your https://github.com/cms-sw/cmssw/files/195815/change_summary.txt


```
error: for class 'TotemRawToDigiStatus' ROOT says the ClassVersion is 6 but classes_def.xml says it is 2. Are you sure everything compiled correctly?
```

Looking in the code
```C++
 +typedef std::map<TotemStructuralVFATId, TotemVFATStatus> TotemRawToDigiStatus; 
```
classVersion should be left out to root in this case, because the type is an STL template.
@Dr15Jones please correct me if I'm wrong.